src/build.sh

```diff
@@ -7,8 +7,13 @@ SRC=""
 #MODE="PROFILE"
 #MODE="MEMCHECK"
 MODE="RELEASE"
+VERSION_FILE="$(cd "$(dirname "$1")" || exit; pwd)/$(basename version.inc)"
```

Comment:
Why not just `VERSION_FILE="version.inc"`? It seems likely to me this script isn't resilient against running in other directories already, if that was the intent, because of all the `$SRC` usages.